#!/usr/bin/env python
PACKAGE = "ros1_sonavision_inference"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("blur_level",     double_t,    0, "Blur level for the input image",        .0, .0, 0.95)
gen.add("darkness_level", double_t,    0, "Darkness level for the input image",    .0, .0, 0.95)

exit(gen.generate(PACKAGE, "ros1_sonavision_inference", "Sonavision"))